py/htm/examples/mnist.py

```diff
@@ -81,7 +81,7 @@ def load_images(file_name):
 default_parameters = {
     'potentialRadius': 7,
     'boostStrength': 7.0,
-    'columnDimensions': (28*28*8, 1),
+    'columnDimensions': (79, 79),
```

Comment:
> SP had dimensions of (X, 1)
Changed to (sqrt(X), sqrt(X))
This helps because the input images are 2-D
Measurements:
 Num Evals | Mean Score | Parameters
        44 |   0.9542   | Before
        20 |   0.9554   | After

I think the changes noted in commit description are only due to random seed fluctuations. As global inh does not have topology, these changes do not matter as long as the num columns is the same(?)

Reply:
I disagree.  The optimization program measured the performance many times, calculated the mean & standard deviation of both distributions, and then calculated the probability that the two distributions were really the same distribution ( AKA the null hypothesis, p = 0.0793 % )